pkg/certs/cert-inspection/certgraphanalysis/metadata_options.go

```diff
@@ -149,6 +152,35 @@ var (
 			return timestampReg.ReplaceAllString(path, "<timestamp>.pem")
 		},
 	}
+	HumanizeRewritePeriod = &metadataOptions{
+		rewriteSecretFn: func(secret *corev1.Secret) {
+			if secret.Annotations == nil {
+				return
+			}
+			if period, ok := secret.Annotations[certrotation.CertificateRefreshPeriodAnnotation]; ok {
+				if d, err := time.ParseDuration(period); err == nil {
+					humanReadableDate := durationToHumanReadableString(d)
+					secret.Annotations[certrotation.CertificateRefreshPeriodAnnotation] = humanReadableDate
+					secret.Annotations[rewritePrefix+"HumanizeRewritePeriod"] = period
```

Comment:
shouldn't this be `rewritePrefix`+`certrotation.CertificateRefreshPeriodAnnotation` ?